WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderActivityLauncher.java

```diff
@@ -86,15 +86,26 @@ public static void showReaderPostPagerForBlog(Context context,
     }
 
     /*
-     * show a list of posts in a specific blog
+     * show a list of posts in a specific blog or feed
      */
     public static void showReaderBlogPreview(Context context, long siteId) {
-        if (siteId == 0) {
+        showReaderBlogPreview(context, siteId, 0);
+    }
+    public static void showReaderBlogPreview(Context context, long siteId, long feedId) {
```

Comment:
Since this method now serves two functions, should the method name be updated to `showReaderBlogOrFeedPreview`? It's also not clear without reading the code that siteId should be 0 if you want to open the feed, and the old method `showReaderFeedPreview(context, feedId)` didn't require a siteId at all so this may cause some confusion. Doing something similar to the way you kept the original `showReaderBlogPreview(context, blogId)` and adding back in `showReaderFeedPreview(context, feedId)` and then just feeding that into the new combined method would go a long way to clear that up.

Reply:
In 08743c9 I renamed the method to `showReaderBlogOrFeedPreview` but I don't think a separate `showReaderFeedPreview` is necessary - at least, I'm not seeing a place where it's needed.